xclim/sdba/detrending.py

```diff
@@ -1,6 +1,7 @@
+# noqa: D205,D400
 """
-Detrending objects
-------------------
+SDBA Detrending Objects
+=======================
```

Comment:
```suggestion
Detrending Objects
==================
```
I think it is obvious enough for the average xclim user ;). Also, the "SDBA" prefix isn't consistently added to all submodules.